health/health.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2024 The VeChainThor developers
+
+// Distributed under the GNU Lesser General Public License v3.0 software license, see the accompanying
+// file LICENSE or <https://www.gnu.org/licenses/lgpl-3.0.html>
+
+package health
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/vechain/thor/v2/thor"
+)
+
+type BlockIngestion struct {
+	BestBlock                   *thor.Bytes32 `json:"bestBlock"`
+	BestBlockIngestionTimestamp *time.Time    `json:"bestBlockIngestionTimestamp"`
```

Comment:
Maybe just `timestamp` ? bestBlockIngestionTimestamp is already wrapped inside blockingestion.

BestBlock could also just be `ID` single the outer struct name already has `Block`

Reply:
Good point ! Making those changes.